src/shared/install.c

```diff
@@ -3461,6 +3471,7 @@ static const char* const unit_file_change_type_table[_UNIT_FILE_CHANGE_TYPE_MAX]
         [UNIT_FILE_UNLINK]      = "unlink",
         [UNIT_FILE_IS_MASKED]   = "masked",
         [UNIT_FILE_IS_DANGLING] = "dangling",
+        [UNIT_FILE_IS_UNKNOWN_TARGET]     = "unknown target",
```

Comment:
re-align the table please

Reply:
Done. But I see a lot of similar tables in other places that are not aligned...